src/js/health-records/containers/Main.jsx

```diff
@@ -179,6 +182,9 @@ export class Main extends React.Component {
     const missingDateError =
     [this.state.startDateError, this.state.endDateError]
       .indexOf('missing') > -1;
+    const startDateError =
+    [this.state.startDateError, this.state.endDateError]
```

Comment:
These could also probably be simplified by creating the error array once. Also, any reason not to use `includes`?